pyperformance/data-files/benchmarks/bm_tomllib_loads/run_benchmark.py

```diff
@@ -0,0 +1,28 @@
+"""
+Benchmark ``loads()`` function of the ``tomllib`` stdlib module
+on a large TOML file of GitHub's real world data.
```

Comment:
Please identify how the data file was generated.  It is useful to be able to reproduce it (or something like it).

Also, is there any PII (personally-identifiable information) in the data that isn't yours or that you don't want exposed?

Reply:
The data was generated by this script https://gist.github.com/kumaraditya303/9b6f76f29e3ac1962320df0c7f9a685a.
and is public issues data of the cpython repo converted to toml.

Reply:
Added link to script.

Reply:
Why not include that code as a script in the benchmark directory?

Reply:
I thought that it would be unnecessary as that script is needed to be run once. However thinking now, I would improve that script to output json and toml in a seprate PR so that the json benchmark can also use real world data instead of dummy data. What do you think?

Reply:
That's good, but it would make sense to start off with the script from the gist as part of this PR.  Even if it's only run once, having the code for that next to the benchmark is still helpful.

Reply:
Okay, will add tomorrow.

Reply:
Added